tycho-p2-director-plugin/src/main/java/org/eclipse/tycho/plugins/tar/TarGzArchiver.java

```diff
@@ -129,6 +129,7 @@ private TarArchiveEntry createTarEntry(File tarRootDir, File source) throws IOEx
             tarEntry.setMode(FilePermissionHelper.toOctalFileMode(attrs.permissions()));
         }
         tarEntry.setModTime(source.lastModified());
+        tarEntry.setCreationTime(null); // tar cannot handle 'LIBARCHIVE.creationtime' attributes 
```

Comment:
This is wrong, e.g. debian supports it: https://manpages.debian.org/unstable/libarchive-dev/tar.5.en.html

Reply:
Understand. I refined it to `GNU tar`. I cannot provide the other details exhaustively.